circle.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/build
+    docker:
+      - image: docker:git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          reusable: true
+          exclusive: true
+      - run:
+          name: Build image
+          command: |
+            docker build -t circleci-docker .
+      - deploy:
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              docker login -u $DOCKER_USER -p $DOCKER_PASS
+              docker tag circleci-docker:latest vlcty/circleci-docker:$CIRCLE_BUILD_NUM
```

Comment:
for consistent naming, use ``-docker`` only on repos, omit from the image names, like the others ``caffe-ubuntu`` image, ``caffe-ubuntu-docker`` repo name

Reply:
Updated. Take a look now!